.github/workflows/changeset-status.yaml

```diff
@@ -0,0 +1,31 @@
+name: Changeset Status
```

Comment:
Why have this in a separate workflow vs another job in the tests workflow?

Reply:
Mainly for ease of having the trigger just be on pull_request rather than also on push to main that the Tests workflow has. We could keep it together and add an extra condition for this job to check for the triggering action.